sqllogictest/src/parser.rs

```diff
@@ -142,9 +143,15 @@ impl Record {
     /// # Panics
     /// If the record is an internally injected record which should not occur in the test file.
     pub fn unparse(&self, w: &mut impl std::io::Write) -> std::io::Result<()> {
+        write!(w, "{}", self)
```

Comment:
I just moved the logic in this function to `impl Display` so that records can formatted with `format!()` as is typical in Rust projects